src/modules/governance/FormattedProposalTime.tsx

```diff
@@ -63,7 +67,8 @@ export function FormattedProposalTime({
       ProposalState.Failed,
       ProposalState.Succeeded,
       ProposalState.Executed,
-    ].includes(state)
+    ].includes(state) &&
+    !l2Execution
   ) {
     return (
       <Typography component="span" variant="caption">
```

Comment:
If proposal is succeeded, button state should be "Passed" with text = "Expected execution on {executionTime}"

Reply:
fixed